include/Knob.h

```diff
@@ -65,6 +65,8 @@ class EXPORT Knob : public QWidget, public FloatModelView
 	mapPropertyFromModel(float,volumeRatio,setVolumeRatio,m_volumeRatio);
 
 	Q_PROPERTY(knobTypes knobNum READ knobNum WRITE setknobNum)
+	
+	Q_PROPERTY(QColor textColor READ textColor WRITE setTextColor)
```

Comment:
https://github.com/LMMS/lmms/blob/master/data/themes/default/style.css#L582

It seems to be reserved for the knob color. Also, it'd be inconsistent with the rest of the codebase.